py_zipkin/thread_local.py

```diff
@@ -16,6 +16,17 @@ def get_thread_local_zipkin_attrs():
     return _thread_local.zipkin_attrs
 
 
+def get_thread_local_span_store():
+    """A wrapper to return _thread_local.span_store
```

Comment:
Would be great to elaborate on the difference between this and the other storage. This one being for output, the other for being for intra-process context propagation